docs/book/src/user/quick-start.md

```diff
@@ -732,19 +735,11 @@ The Docker provider is not designed for production use and is intended for devel
 
 </aside>
 
```

Comment:
A compromise overall could be to say: if you need upgrades to work use development-topology otherwise use development

Reply:
The compromise seems fair to me

* having the `development` flavor as second option
* add the note

Reply:
>if you need upgrades to work use development-topology otherwise use development

That seems confusing from my POV. Maybe we could make the topology version the default version 'development' and change the current non-ClusterClass to 'deployment-legacy', leave it in the guide and add a note about upgrades not working correctly with the legacy version.